doc/adr/0015-git-branching-strategy.md

```diff
@@ -0,0 +1,31 @@
+# 15. Git branching strategy
+
+Date: 2022-11-28
+
+## Status
+
+Accepted
+
+## Context
+
+During development, there are different git branches in our repository. When merging them, there are multiple possibilities, so for a clean repository, there is a decision needed.
+
+See also the ADR for the corresponding app: [useid-app-ios/0003-branching-strategy](https://github.com/digitalservicebund/useid-app-ios/blob/e76dababb2015fa05ddabb60a29827892502dad0/doc/adr/0003-branching-strategy.md)
+
+## Decision
+
+As our main development branch we use the `main` branch (there is currently no need for a separate `develop` branch). We strive for keeping a linear git history on those branches if possible ([A tidy, linear Git history](https://www.bitsnbites.eu/a-tidy-linear-git-history/) for some rationale), therefore we use fast-forward merges instead of merge commits when merging pull requests.
```

Comment:
```suggestion
As our main development branch we use the `main` branch. We strive for keeping a linear git history on this branch if possible ([A tidy, linear Git history](https://www.bitsnbites.eu/a-tidy-linear-git-history/) for some rationale), therefore we use fast-forward merges instead of merge commits when merging pull requests.
```